src/services/WalletService.ts

```diff
@@ -53,7 +53,11 @@ export default class WalletService {
     return balance ? parseInt(balance) / 1e18 : 0;
   }
 
-  static async connectSuperHero(): Promise<string> {
+  static getAeBalance(address: `ak_${string}`): Promise<string> {
+    return aeSdk.getBalance(address).catch((error) => {console.info(error); return '0';});
```

Comment:
I suggest to split lines 
```suggestion
    return aeSdk.getBalance(address).catch((error) => {
      console.info(error);
      return '0';
    });
```